docs/developer-guide/coordinate-systems.md

```diff
@@ -19,42 +17,69 @@ new PointCloudLayer({
     {position: [33.22, 109.87, 1.455]}, // meter offsets from the coordinate origin
     ...
   ],
-  ...
+  radiusPixels: 2,
+  sizeUnits: 'pixels'
 })
 ```
 
 
-## Supported Coordinate Systems
+## Concepts
 
-| Coordinate System Mode               | Positions                   | Dimensions | Notes |
-| ---                                  | ---                           | --- | --- |
-| `COORDINATE_SYSTEM.LNGLAT` (default) | [longitude, latitude, altitude] | meters | Longitude and latitude are specified as [WGS84](https://gisgeography.com/wgs84-world-geodetic-system/) coordinates in degrees from Greenwich meridian / equator respectively, and altitude is specified in meters above sea level. |
-| `COORDINATE_SYSTEM.METER_OFFSETS` *   | [Δx, Δy, Δz]   | meters | Positions are given in meter offsets from a reference geo-location that is specified separately (`coordinateOrigin`). The `x` axis points map east, the `y` axis points map north, and `z` points up. |
-| `COORDINATE_SYSTEM.LNGLAT_OFFSETS`    | [Δlongitude, Δlatitude, Δaltitude]   | meters | Positions are given in meter offsets from a reference geo-location that is specified separately (`coordinateOrigin`). |
-| `COORDINATE_SYSTEM.CARTESIAN`         | [x, y, z] | identity units | A linear system with no interpretation for pure info-vis layers. Viewports can be used without supplying geospatial reference points. |
-| `COORDINATE_SYSTEM.LNGLAT_DEPRECATED`| [longitude, latitude, altitude] | meters | A lower precision version of the `COORDINATE_SYSTEM.LNGLAT` mode, that was the default until deck.gl v6.2. Will be removed in a future release. |
+Throughout this document you will find reference to the following **coordinate spaces** in deck.gl's projection system:
+
+##### World space
+
+The natural coordinate system of a dataset. It is usually determined by the source of the data, like the device it is generated on, and the format that it is stored as on disk or a cloud data warehouse.
+
+Some examples of different world space definitions include a [Shapefile](https://en.wikipedia.org/wiki/Shapefile) with any [standard geospatial reference](https://epsg.io/); WGS84 as used by GeoJSON or KML; GPS traces; LIDAR scans; info-vis values that do not map to real-world dimensions; [Slippy Map](https://wiki.openstreetmap.org/wiki/Slippy_Map) tiles from services like OpenStreetMaps and Mapbox; 3D tiling standards such as [Cesium 3D Tiles](https://www.ogc.org/standards/3DTiles) and [Esri I3S](https://www.ogc.org/standards/i3s).
+
+Positions in different world spaces come with different scales and orientations, may be measured relative to static or dynamic reference points, and in many cases are not linear. It is common for an application to overlay multiple datasets from different world spaces into the same 3D view.
```

Comment:
```suggestion
Positions in different world spaces come with different scales and orientations, may be measured relative to static or dynamic reference points, and in many cases their scales are not linear. It is common for an application to overlay multiple datasets from different world spaces into the same 3D view.
```
Not sure what the best wording is, but the existing `in many cases are not linear` seemed to be referencing `Positions` which didn't seem right to me.